lib/RTCSession.js

```diff
@@ -1055,14 +1055,7 @@ module.exports = class RTCSession extends EventEmitter
       return;
     }
 
-    if (this._tones)
-    {
-      // Tones are already queued, just add to the queue.
-      this._tones += tones;
-
-      return;
-    }
-
```

Comment:
This must remain as it it, because there may already be tones being sent by other previous calls to this method.

What we want to do is:
- remove the variable `position`. It's local to this function and breaks things when there are already tones being sent.
- substitute code above
```js
  const tone = this._tones[position];
  position += 1;
```

by
```js
// Retrieve the next tone.
const tone = this._tones[0];
 
// Remove the tone from this._tones.
this._tones.substring(1);
```